code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -733,7 +759,11 @@
 		// Overall, gamma borg has higher skill floor but lower skill ceiling.
 		/obj/item/melee/baton/loaded, // secondary weapon, for things immune to burn, immune to ranged weapons, or for arresting low-grade threats
 		/obj/item/restraints/handcuffs/cable/zipties/cyborg,
-		/obj/item/pickaxe/drill/jackhammer // for breaking walls to execute flanking moves
+		/obj/item/pickaxe/drill/jackhammer, // for breaking walls to execute flanking moves
+		/obj/item/gripper/nuclear,
+		/obj/item/crowbar/cyborg,
+		/obj/item/gps/cyborg,
+		/obj/item/pinpointer/operative/nad,
```

Comment:
Again, operative pinpointer, usually you won't see an ERT during a nukie round but they probably shouldn't have it